Include/internal/pycore_unicodeobject.h

```diff
@@ -247,6 +247,12 @@ extern Py_ssize_t _PyUnicode_InsertThousandsGrouping(
     Py_UCS4 *maxchar,
     int forward);
 
+/* Dedent a string.
+   Behaviour is expected to be an exact match of `textwrap.dedent`.
+   Return a new reference on success, NULL with exception set on error.
+   */
+PyAPI_FUNC(PyObject*) _PyUnicode_Dedent(PyObject *unicode);
```

Comment:
Is the export needed as a PyAPI_FUNC needed for `Modules/main.c`? can't we live with `extern` only?

Reply:
I can't tell the difference here. But I can hide it for now.